src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -723,13 +718,6 @@ class DocBaseViewer extends BaseViewer {
         // Disable streaming by default unless it is explicitly enabled via options
         const disableStream = this.getViewerOption('disableStream') !== false;
 
-        // If range requests and streaming are disabled, request the gzip compressed version of the representation
-        this.encoding = disableRange && disableStream ? ENCODING_TYPES.GZIP : undefined;
-
-        if (this.encoding) {
-            queryParams[QUERY_PARAM_ENCODING] = this.encoding;
```

Comment:
I think we can delete `queryParams` and remove the `appendQueryParams` call now, since it's a noop.